toqito/nonlocal_games/nonlocal_game.py

```diff
@@ -5,11 +5,74 @@
 
 import cvxpy
 import numpy as np
+import numba as nb
 
 from toqito.helper import npa_constraints, update_odometer
 from toqito.matrix_ops import tensor
 from toqito.rand import random_povm
 
+from binary_constraint_system_game import check_perfect_commuting_strategy,tensor_to_raw
+
+@nb.njit
+def _fast_classical_value(pred_mat: np.ndarray, num_b_out: int, num_b_in: int, pow_arr: np.ndarray):
+    r"""Compute the classical winning probability by iterating over all deterministic strategies.
+    Expects pred_mat shaped (A_out, A_in, B_out, B_in),
+    where B_out == num_b_out and B_in == num_b_in.
+    Parameters
```

Comment:
Can you please update the docstrings here to abide by the same stylistic convention that other functions in toqito use?